tests/cli/test_suite.py

```diff
@@ -1281,138 +2165,419 @@ def test_suite_scaffold_on_existing_suite_raises_error(
+
+    suite: ExpectationSuite = context.get_expectation_suite(
+        expectation_suite_name=expectation_suite_name
+    )
+    assert suite.expectations == [
+        ExpectationConfiguration(
+            **{
+                "expectation_type": "expect_table_columns_to_match_ordered_list",
+                "kwargs": {
+                    "column_list": [
+                        "Unnamed: 0",
+                        "Name",
+                        "PClass",
+                        "Age",
+                        "Sex",
+                        "Survived",
+                        "SexCode",
+                    ]
+                },
+                "meta": {},
+            }
+        ),
+        ExpectationConfiguration(
+            **{
+                "expectation_type": "expect_table_row_count_to_be_between",
+                "kwargs": {"max_value": 1313, "min_value": 1313},
+                "meta": {},
+            }
+        ),
     ]
-    assert mock_emit.call_count == 2
+
+    assert mock_subprocess.call_count == 0
+
+    assert mock_webbroser.call_count == 0
+
+    assert mock_emit.call_count == 5
     assert mock_emit.call_args_list == [
         mock.call(
             {"event_payload": {}, "event": "data_context.__init__", "success": True}
         ),
         mock.call(
-            {"event": "cli.suite.scaffold", "event_payload": {}, "success": True}
+            {
+                "event": "cli.suite.new.begin",
+                "event_payload": {"api_version": "v3"},
+                "success": True,
+            }
+        ),
+        mock.call(
+            {
+                "event_payload": {
+                    "anonymized_expectation_suite_name": "9df638a13b727807e51b13ec1839bcbe"
+                },
+                "event": "data_context.save_expectation_suite",
+                "success": True,
+            }
+        ),
+        mock.call(
+            {
+                "event": "cli.suite.new.end",
+                "event_payload": {"api_version": "v3"},
+                "success": True,
+            }
+        ),
+        mock.call(
+            {"event_payload": {}, "event": "data_context.__init__", "success": True}
         ),
     ]
+
     assert_no_logging_messages_or_tracebacks(
         my_caplog=caplog,
         click_result=result,
-        allowed_deprecation_message=VALIDATION_OPERATORS_DEPRECATION_MESSAGE,
     )
 
 
-@pytest.mark.xfail(
-    reason="This command is not yet implemented for the modern API",
-    run=True,
-    strict=True,
-)
 @mock.patch(
     "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
 )
 @mock.patch("subprocess.call", return_value=True, side_effect=None)
-def test_suite_scaffold_creates_notebook_with_no_jupyter_flag(
-    mock_subprocess, mock_emit, caplog, monkeypatch, titanic_data_context_stats_enabled
+@mock.patch("webbrowser.open", return_value=True, side_effect=None)
+def test_suite_new_profile_creates_notebook_opens_jupyter(
```

Comment:
How about changing the name of this test since it runs the notebook and generates a suite?
Also these tests are also a prime candidate to parametrize. Not necessary for now just a thought for future refactor.

Reply:
Name changed (in the next commit).  Parameterization should be in a separate PR.